README.md

```diff
@@ -74,13 +74,13 @@ then
 yarn serve
 ```
 
-## Project setup for vscode and docker (developing inside dev container) on linux
+## Project setup for vscode and docker (developing inside dev container) on Linux
 
-The goal is it have a complete dev environment very quickly and isolated.
+The goal is it has a complete dev environment very quickly and isolated.
```

Comment:
i actually think the typo here is "it"...
otherwise it's a rather strange sentence ... (before AND after) :D 

```suggestion
The goal is to have a complete dev environment very quickly and isolated.
```